### DISCUSSION TITLE
Non-invertible matrix in interpolation

### DISCUSSION THREAD

**Discussion by lily-dove** [GliderToolsCommunity/GliderTools#153] [Q&A]
<img width="372" alt="Screen Shot 2022-07-22 at 3 37 54 PM" src="https://user-images.githubusercontent.com/66711366/180579535-a02c5fc1-f011-434e-bf00-38a964ad21b9.png">
Hi all! My summer student and I are using GliderTools for a project. It's been wonderful so far! We've come across a challenge where we're trying to do the horizontal interpolation for chunks of ~50000 datapoints. It's worked for several chunks of data but we've hit one where halfway through the interpolation, we get a "LinAlgError: singular matrix" error. I'll post the full error message as an attachment. I'd appreciate any insight y'all have on how to go about troubleshooting this...

Thank you!

**Comment by soerenthomsen** [comment]
Hi @lily-dove thanks for raising this error in the discussion. I transferred it into an issue.
